tools/ci_build/github/linux/docker/scripts/install_ubuntu.sh

```diff
@@ -67,6 +67,7 @@ fi
 
 /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall pip==19.0.3
 /usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall numpy==1.15.0
+/usr/bin/python${PYTHON_VER} -m pip install --upgrade --force-reinstall requests==2.21.0
```

Comment:
This is for ONNX Runtime server integration tests. It could not be installed in the build.py due to the access right limitation of current user. It should not impact other Python packages and the rest of build.